R/oauth_wrapper.R

```diff
@@ -0,0 +1,24 @@
+#' @export
+CopiedToken2.0 <- R6::R6Class("CopiedToken2.0", inherit = httr::Token2.0, list(
```

Comment:
When in exploratory package's context, I think this should be called something like HttrOAuthToken2.0, since being about httr's OAuth is not an obvious thing here.